docs/data-cookbooks/setup.md

```diff
@@ -0,0 +1,103 @@
+
+**Note:** The schemas are _extremely_ useful for understanding what attributes are configurable and which of those are required. Indeed, they are what the Cumulus code validates definitions (whether that be collection, provider, or others) against. Much of this document is simply providing some context to the information in the schemas.
+
+
+### Collections
+
+Collections are logical sets of data objects of the same data type and version. A collection provides contextual information used by Cumulus ingest. We have a few [test collections](https://github.com/nasa/cumulus/tree/master/example/data/collections) configured in Cumulus source for integration testing. Collections can be viewed, edited, added, and removed from the Cumulus dashboard under the "Collections" navigation tab. Additionally, they can be managed via the [collections api](https://nasa.github.io/cumulus-api/?language=Python#list-collections).
+
+The schema for collections can be found [here](https://github.com/nasa/cumulus/tree/master/packages/api/models/schemas.js) as the object assigned to `module.exports.collection` and tells us all about what values are expected, accepted, and required in a collection object (where required attribute keys are assigned as a string to the `required` attribute).
+
+**Break down of [s3_MOD09GQ_006.json](https://github.com/nasa/cumulus/tree/master/example/data/collections/s3_MOD09GQ_006.json)**
+
+|Key  |Value|Required  |Description|
+|:---:|:-----:|:--------:|---|
+|name |`"MOD09GQ"`|Yes|The name attribute designates the name of the collection. This is the name under which the collection will be displayed on the dashboard|
+|version|`"006"`|Yes|A version tag for the collection|
+|process|`"modis"`|Yes|The options for this are found in "ChooseProcess and in workflows.yml|
+|granuleId|`"^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}$"`|Yes|REGEX to match granuleId|
+|granuleIdExtraction|`"(MOD09GQ\\..*)(\\.hdf\|\\.cmr\|_ndvi\\.jpg)"`|Yes|REGEX that extracts granuleId from file names|
+|sampleFileName|`"MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"`|Yes|...|
+|files|`<JSON Object>` of files defined [here](#files)|Yes|Describe the individual files that will exist for each granule in this collection (size, browse, meta, etc.)|
+|provider_path|`"cumulus-test-data/pdrs"`|No|This collection is expecting to find data in a `cumulus-test-data/pdrs` directory, whether that be in S3 or at an http endpoint|
+|dataType|`"MOD09GQ"`|No|# TODO|
+|duplicateHandling|`"replace"`|No|(replace\|version\|skip) determines granule duplicate handling scheme|
+|url_path|`"{cmrMetadata.Granule.Collection.ShortName}/{substring(file.name, 0, 3)}"`|No|Filename without extension|
```

Comment:
This line is making this table look really weird. Not really sure what to do about it... if you could put different parts on separate lines to make this column width smaller.

Reply:
Yes, that's kind of an awkward formatting. I was hoping that it would work for wider screens, but that's maybe not the best way to think about it. What do you think about something like:

|key|value|required|description|
|---|---|---|---|
|**type1** url_path|`"{cmrMetadata.Granule.Collection.ShortName}/`<br />`{substring(file.name, 0, 3)}"`|No|Filename without extension|
|**type2** url_path|`"${collectionShortName}/{substring(file.name, 0, 3)}"`<br />Where: <br />`${collectionShortName} = {cmrMetadata.Granule.Collection.ShortName}`|No|Filename without extension|

Reply:
I lean more towards **type1** because it seems to be more robust...